plugins/tiddlywiki/highlight/highlightblock.js

```diff
@@ -16,9 +16,23 @@ var TYPE_MAPPINGS_BASE = "$:/config/HighlightPlugin/TypeMappings/";
 
 var CodeBlockWidget = require("$:/core/modules/widgets/codeblock.js").codeblock;
 
-var hljs = require("$:/plugins/tiddlywiki/highlight/highlight.js");
+var useNew = true;
```

Comment:
What's the reason to have this new global variable?

Reply:
To determine which version of highlight.js is loaded so CodeBlockWidget can use the correct API calls.